db/migrate/20170129075432_create_resources_table.rb

```diff
@@ -0,0 +1,11 @@
+class CreateResourcesTable < ActiveRecord::Migration
+  def change
+    create_table :resources do |t|
+      t.string  :name
+      t.string  :description
```

Comment:
I would say description should be a text and not a string as it could be longer.